crates/resolc/benches/compile.rs

```diff
@@ -0,0 +1,183 @@
+use std::{
+    fs::File,
+    path::PathBuf,
+    process::{Command, Output, Stdio},
+    time::Duration,
+};
+
+use criterion::{
+    criterion_group, criterion_main,
+    measurement::{Measurement, WallTime},
+    BatchSize, BenchmarkGroup, Criterion,
+};
+use resolc::{self, SolcCompiler};
+
+/// The function under test executes the `resolc` executable.
+fn execute_resolc(arguments: &[&str], stdin_config: Stdio) {
+    execute_command(resolc::DEFAULT_EXECUTABLE_NAME, arguments, stdin_config);
+}
+
+/// The function under test executes the `solc` executable.
+fn execute_solc(arguments: &[&str], stdin_config: Stdio) {
+    execute_command(
+        SolcCompiler::DEFAULT_EXECUTABLE_NAME,
+        arguments,
+        stdin_config,
+    );
+}
+
+#[inline(always)]
```

Comment:
```suggestion
```

Is this intentional? We don't blindly sprinkle the code with compiler hints.

Reply:
That's a valid thought :) However consider the impact negligible and you don't know whether the compiler does inline this anyways (and the compiler heuristics are almost always better than what humans think anyways). So these are almost always unnecessary.

Reply:
Ah okay, thanks :)